### PR TITLE
Fixed date parsing (addresses site not working bug)

### PR DESCRIPTION
**Pull request by geometrikal**
Site was not loading, with error thrown in parseISO. It was found that Xively already returned the date in the correct format so the parsing of the date in parseISO is unnecessary and causes the bug. The parsing code was commented out.

### CODE REVIEW

File: follows.js [hunk 1]
```diff
@@ -59,26 +59,28 @@
...
-		if(z && !/Z$/.test(iso)){
-			TZ=/:\d\d((\-|\+)(\d\d):(\d\d))$/.exec(iso);
-			if(TZ){
-				tem= TZ[3]*60+(+TZ[4]);
-				z+= TZ[2]+tem;
-				iso= iso.replace(TZ[1],'');
-			}
-		}
-		iso= iso.replace(/[^\d:]/g,' ')+z;
+		//var z, tem, TZ, ms= 0;
```
Comment (smulube) [inline follows.js]: @geometrikal - could this code just be removed rather than commented out?

File: follows.js [hunk 2]
```diff
@@ -59,26 +59,28 @@
...
+		//		z+= TZ[2]+tem;
+		//		iso= iso.replace(TZ[1],'');
+		//	}
+		//}
+		//iso= iso.replace(/[^\d:]/g,' ')+z;
 		var stamp= Date.parse(iso);
 		if(!stamp) throw iso +' Unknown date format';
-		return new Date(stamp+ms);
+		//return new Date(stamp+ms);
+        return new Date(stamp);
```
Comment (errordeveloper) [inline follows.js]: @geometrikal please keep indentation consistent with the rest of the code.

File: follows.js (hunk 1 above)
Comment (errordeveloper) [inline follows.js]: I certainly agree with @smulube - we should not have bits of commented code, as we can alway look in the git history if we need to refer to anything.

**Comment by geometrikal** [comment]
Have updated as per comments. Tab error was Xcode.. Now using komodo instead :)

**Pull request merged**